packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -4515,6 +4518,7 @@ export interface VideoGalleryProps {
 
 // @public
 export interface VideoGalleryRemoteParticipant extends VideoGalleryParticipant {
+    contentSharingStream?: HTMLElement;
```

Comment:
This should be of type `VideoGalleryStream` to keep consistency with other streams